components/pd_client/src/client.rs

```diff
@@ -140,31 +140,48 @@ impl RpcClient {
     }
 
     /// Gets given key's Region and Region's leader from PD.
```

Comment:
It looks so inconsistent that some functions are async while some others are blocking, and they all look similar in function names...